server/src/main/java/io/spine/server/model/MethodPredicate.java

```diff
@@ -35,11 +35,11 @@
 public abstract class MethodPredicate implements Predicate<Method> {
 
     @Override
-    public boolean apply(@Nullable Method method) {
+    public boolean test(@Nullable Method method) {
         checkNotNull(method);
         boolean result = verifyAnnotation(method)
-                         && verifyParams(method)
-                         && verifyReturnType(method);
+                && verifyParams(method)
+                && verifyReturnType(method);
```

Comment:
Seems to be an accidental change. Please undo.

Reply:
It's the current IDEA layout.